specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2023-09-01-preview/resourcegraph.json

```diff
@@ -0,0 +1,602 @@
+            }
+          },
+          "default": {
+            "description": "An error occurred while processing the request. See the error.code parameter to identify the specific error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "QueryRequest": {
+      "description": "Describes a query to be executed.",
+      "type": "object",
+      "properties": {
+        "subscriptions": {
+          "description": "Azure subscriptions against which to execute the query.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Subscription Id."
+          }
+        },
+        "managementGroups": {
+          "description": "Azure management groups against which to execute the query. Example: [ 'mg1', 'mg2' ]",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "The name of Management group."
+          }
+        },
+        "query": {
+          "description": "The resources query.",
+          "type": "string"
+        },
+        "options": {
+          "description": "The query evaluation options",
+          "$ref": "#/definitions/QueryRequestOptions"
+        },
+        "facets": {
+          "description": "An array of facet requests to be computed against the query result.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FacetRequest"
+          }
+        }
+      },
+      "required": [
+        "query"
+      ]
+    },
+    "QueryRequestOptions": {
+      "description": "The options for query evaluation",
+      "properties": {
+        "$skipToken": {
+          "description": "Continuation token for pagination, capturing the next page size and offset, as well as the context of the query.",
+          "type": "string"
+        },
+        "$top": {
+          "description": "The maximum number of rows that the query should return. Overrides the page size when ```$skipToken``` property is present.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1,
+          "maximum": 1000
+        },
+        "$skip": {
+          "description": "The number of rows to skip from the beginning of the results. Overrides the next page offset when ```$skipToken``` property is present.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0
+        },
+        "resultFormat": {
+          "description": "Defines in which format query result returned.",
+          "type": "string",
+          "enum": [
+            "table",
+            "objectArray"
+          ],
+          "default": "objectArray",
+          "x-ms-enum": {
+            "name": "ResultFormat",
+            "modelAsString": false
```

Comment:
I'm trying to remember if this needs to be modelAsString true, if you want it to be extensible in the future...

Reply:
Hi Tim, this swagger is not updated, and this definition existed before.

Reply:
Only resourcecopilot.json is newly added here.

Reply:
Well you can't change from string without it being SDK breaking so probably simpler not to fix this ;(